README.md

```diff
@@ -8,16 +8,40 @@
 
 _IMPORTANT it still in development and it's not fully functional_
 
+This library is based on [openid-4-verifiable-credential-issuance](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) for requesting Verifiable Credentials.
+
+### Flow
+#### Pre-authorized Code Flow
+The below diagram shows the steps involved in this flow. Note that wallet inner functionalities (like saving VCs) are out of scope of this library. Also This library doesn't involve any functionalities of a VC Issuer
+![Flow diagram](https://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/Sphereon-Opensource/OIDC4VCI-client/develop/docs/preauthorized-code-flow.puml)
+
 ### Interfaces
 
 ```typescript
 export interface IssuanceInitiationRequestPayload {
   issuer: string, //REQUIRED The issuer URL of the Credential issuer, the Wallet is requested to obtain one or more Credentials from.
-  credential_type: string[] | string, //REQUIRED A JSON string denoting the type of the Credential the Wallet shall request
+  credential_type: string[], //REQUIRED A JSON string denoting the type of the Credential the Wallet shall request
```

Comment:
Why is the singular string removed? The actual example in the spec uses a singular string for it

Reply:
Fixed